datajoint/autopopulate.py

```diff
@@ -28,8 +53,9 @@ def key_source(self):
         """
         :return: the query expression that yields primary key values to be passed,
         sequentially, to the ``make`` method when populate() is called.
-        The default value is the join of the parent relations.
-        Users may override to change the granularity or the scope of populate() calls.
+        The default value is the join of the parent tables references from the primary key.
+        Subclasses may override they key_source to change the scope or the granularity 
```

Comment:
`datajoint/autopopulate.py:57:87: W291 trailing whitespace`

The style test failed